bundles/org.openhab.binding.konnected/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -148,4 +160,252 @@
+				<label>Momentary</label>
+				<description>The duration of the pulse in millisecods</description>
+			</parameter>
+			<parameter name="pause" type="integer">
+				<label>Pause</label>
+				<description>The time between pulses in millisecods</description>
+			</parameter>
+			<parameter name="times" type="integer">
+				<label>Times</label>
+				<description>is the number of times to repeat or `-1` for an infinitely repeating pulse</description>
+			</parameter>
+		</config-description>
+	</channel-type>
+	<channel-type id="temperature-pro">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>This zone measures temperature</description>
+		<state readOnly="true"/>
+		<config-description>
+			<parameter name="zone" type="text">
+				<label>Zone Number</label>
+				<description>The Zone Number of the channel.</description>
+				<options>
+					<option value="1">1</option>
+					<option value="2">2</option>
+					<option value="3">3</option>
+					<option value="4">4</option>
+					<option value="5">5</option>
+					<option value="6">6</option>
+					<option value="7">7</option>
+					<option value="8">8</option>
+				</options>
+			</parameter>
+			<parameter name="tempsensorType" type="boolean">
+				<label>DHT22</label>
+				<description>Is the sensor a dht22 or a ds18b20? Set to true for dht22 sensor</description>
+			</parameter>
+			<parameter name="pollinterval" type="integer">
+				<label>Poll Interval</label>
+				<description>The interval in minutes to poll the sensor.</description>
+			</parameter>
+			<parameter name="ds18b20_address" type="text">
+				<label>DS18b20 Address</label>
+				<description>This is the unique address of the sensor on the one wire bus.</description>
+			</parameter>
+		</config-description>
+	</channel-type>
+	<channel-type id="humidity-pro">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>This zone measures humidity</description>
+		<state readOnly="true"/>
+		<config-description>
+			<parameter name="zone" type="text" required="true">
+				<label>Zone Number</label>
+				<description>The Zone Number of the channel.</description>
+				<options>
+					<option value="1">1</option>
+					<option value="2">2</option>
+					<option value="3">3</option>
+					<option value="4">4</option>
+					<option value="5">5</option>
+					<option value="6">6</option>
+					<option value="7">7</option>
+					<option value="8">8</option>
+				</options>
+			</parameter>
+		</config-description>
+	</channel-type>
+	<channel-type id="actuator-pro-alarm1">
+		<item-type>Switch</item-type>
+		<label>Alarm 1</label>
+		<description>The Alarm 1 Output on the Konnected Pro</description>
+		<config-description>
+			<parameter name="zone" type="text" required="true">
+				<label>Zone Number</label>
+				<description>The Zone Number of the channel.</description>
+				<default>13</default>
+				<options>
+					<option value="13">Output1</option>
```

Comment:
Is there only one option?

Reply:
For this one yes.  This is a special channel that will always be present as it is not configurable by the user.  However it needs to be set to 13 for this channel for processing in the binding based on the zone number.  (see line 57 of KonnectedModeuleGSON for an example)